yureks-news/src/App.js

```diff
@@ -17,6 +17,7 @@ function App() {
           <Route path="/" element={<MainPage />} />
           <Route path="/articles" element={<AllArticles />} />
           <Route path="/articles/:article_topic" element={<TopicArticles />} />
```

Comment:
I would put these differently...

/topics/:article_topic - TopicArticles
/articles/:article_id - IndividualArticle

This would make for more readable routing.